gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -309,6 +309,12 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-text-disabled">
               The standard color for text within disabled elements in RSC.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-text-placeholder">
+              The standard color for placeholder text within a text input.
+            </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-input-text">
+              The text color used for the <NxCode>NxTextInput</NxCode> value.
```

Comment:
I wonder if instead of specifying NxTextInput specifically we could just keep it general, like text color used inside input field or something like that. 
This is assuming that other components that use TextInput under the hood also have the same color: NxDateInput, NxComboBox, etc.